.github/workflows/cargo-audit.yml

```diff
@@ -11,10 +11,19 @@ on:
       - '**/Cargo.toml'
       - '**/Cargo.lock'
 jobs:
-  audit:
+  cargo-deny:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        checks:
+          - advisories
+          - bans licenses sources
+
+    # Prevent sudden announcement of a new advisory from failing ci:
+    continue-on-error: ${{ matrix.checks == 'advisories' }}
```

Comment:
I don't read the warnings whenever the overall run succeeds. Thus I would have missed https://github.com/libp2p/rust-libp2p/issues/2794 right? How about failing instead? I know it is more noise, but I would argue it is worth it.

Reply:
Hm. I was going to ask whether we should keep the cron job and it should be kept separate and PR still has checks for licenses and other things - the cron - a separate workflow action - would still raise separately Issue about any advisories - or do we want to keep the CI failing in this case?

Reply:
So I changed now that the PR is now on cargo deny upon PR and will not fail straight from advisories whilst there is still periodic cron job to raise any advisory failures separately via audit - the periodic schedule can be always changed